<launch>
    <arg name="camera_namespace" default="camera_side"/>

    <node name="laplacian_line_detector" pkg="rr_iarrc" type="laplacian_line_detector"
          output="screen" ns="$(arg camera_namespace)">
        <!--Range of the Laplacian Area to Floodfill-->
        <param name="laplacian_threshold_min" type="int" value="-4" />
        <param name="laplacian_threshold_max" type="int" value="5" />
        <!--Minimum Area to Keep-->
        <param name="min_blob_area" type="int" value="200" />

        <!--Strength of Adaptive Thresholding-->
        <param name="ignore_adaptive" type="bool" value="True" />

        <param name="subscription_node" type="string" value="image_raw"/>
        <!-- <param name="subscription_node" type="string" value="image_raw"/> --> <!-- this allows it to run in sim-->
        <param name="publish_detection_node" type="string" value="lines/detection_img"/>
        <param name="publish_debug_node" type="string" value="lines/debug_img"/>
    </node>
</launch>
